cogs/meta.py

```diff
@@ -23,29 +56,25 @@ async def check(self, ctx):
         except Exception as e:
             await ctx.send('Seems like I\'m getting cloud flared, rip. ' +
                            str(e))
-
     @commands.command()
     async def stats(self, ctx):
-        """Display cool dmoj stats that no one asked for"""
```

Comment:
Could we keep the old command and move it back from misc?

Reply:
reverted changes, though if we added more misc commands it should go in its own category because it doesn't have much to do with bot meta